frameworks/helloworld/src/test/java/com/mesosphere/sdk/helloworld/scheduler/CustomStepsTest.java

```diff
@@ -123,6 +123,16 @@ public void testSerialStrategyParallelSteps() throws Exception {
                 .run(ticks);
     }
 
+    @Test(expected = IllegalStateException.class)
+    public void testInvalidStep() throws Exception {
+        Collection<SimulationTick> ticks = new ArrayList<>();
+        new ServiceTestRunner("custom_steps.yml")
+                .setSchedulerEnv(
+                        "DEPLOY_STRATEGY", "serial",
+                        "DEPLOY_STEPS", "[[foo, bar]]")
+                .run(ticks);
```

Comment:
can replace this with `.run(ticks);` with `.run(new ArrayList<>());` and then delete line 128.

Reply:
- Could use `Collections.emptyList()` instead of creating a new arraylist (one difference is that the collections one is immutable)
- Should be able to just call the `run()` overload